constants/limit.go

```diff
@@ -6,6 +6,15 @@ package constants
 
 // Limits and constraints.
 const (
+	// BuildLimitMin defines the minimum value for repo concurrent build limit.
+	BuildLimitMin = 1
+
+	// BuildLimitMax defines the maximum value for repo concurrent build limit.
+	BuildLimitMax = 30
+
+	// BuildLimitDefault defines the default value for repo concurrent build limit.
+	BuildLimitDefault = 10
```

Comment:
Do people feel these values are good in their current state?

Reply:
I'm good with these being the default given the fact that we will allow the administrator to override them of the server if they want.